core/unit_test/TestDeepCopy.hpp

```diff
@@ -2,6 +2,7 @@
 
 namespace Test {
 
+#ifdef KOKKOS_ENABLE_CXX11_DISPATCH_LAMBDA
```

Comment:
This feels orthogonal to the intent of the PR. I'll still merge, but want to make sure I'm not missing some implementation detail

Reply:
Answered via Slack, this was moved from where the test was included to surround the test. That's a lot more informative to me